.github/workflows/build.yml

```diff
@@ -17,36 +17,35 @@ jobs:
 
       - name: Set Environment Variables
         run: |
+               echo >> $GITHUB_ENV POSTGRES_USER='prime'
+               echo >> $GITHUB_ENV POSTGRES_PASSWORD='changeIT!'
+               
                if [ "$GITHUB_BASE_REF" == "production" ]
                then
                    echo "Building for the production environment."
                    echo >> $GITHUB_ENV ACR_REPO=pdhprodcontainerregistry.azurecr.io
                    echo >> $GITHUB_ENV PREFIX=pdhprod
-                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_PROD_USER }}
-                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_PROD_PWD }}
                else
                    echo "Building for the test environment."
                    echo >> $GITHUB_ENV ACR_REPO=pdhtestcontainerregistry.azurecr.io
                    echo >> $GITHUB_ENV PREFIX=pdhtest
-                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_TEST_USER }}
-                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_TEST_PWD }}
                fi
 
       - name: Setup PostgreSQL
         uses: Harmon758/postgresql-action@0be19fa37850b22cb4c9bbf28a03abbf44abd863
         with:
           postgresql version: 11
           postgresql db: prime_data_hub
-          postgresql user: ${{ env.POSTGRESQL_USER }}
-          postgresql password: ${{ env.POSTGRESQL_PWD }}
+          postgresql user: ${{ env.POSTGRES_USER }}
+          postgresql password: ${{ env.POSTGRES_PASSWORD }}
```

Comment:
Great find.

Reply:
@Mr0grog - could you revert this as well per the comment above?

Reply:
Oh, wait, I missed this comment and am a little confused. I understand reinstating secrets (https://github.com/CDCgov/prime-data-hub/pull/271#discussion_r561079751), but if we don’t keep this change (and the one to the `mvn package` line below), these secrets HAVE to be the same as the passwords that are plaintext elsewhere in the codebase (otherwise the build will fail here), which seems to defeat the purpose of having secrets. Why are we changing this?

Reply:
Sorry, you are correct.  I missed the connection between the maven build and this.